src/store/store.js

```diff
@@ -174,13 +174,17 @@ class Store {
       if (obj.some (x => x.offset === undefined)) {
```

Comment:
@SL you can move the `if (obj.some (...)` code down into the `if (obj.length > 0)` code block.